privacyidea/lib/tokens/u2ftoken.py

```diff
@@ -291,10 +301,12 @@ def update(self, param, reset_failcount=True):
         TokenClass.update(self, param)
         description = "U2F initialization"
         reg_data = getParam(param, "regdata")
+        verify_cert = is_true(getParam(param, "u2f.verify_cert", True))
```

Comment:
Hmm, if "u2f.verify_cert" is not in ``param``, ``getParam`` returns None here. Should it instead default to True?

Reply:
I think it does, doesn't it? Oh, it should be

     verify_cert = is_true(getParam(param, "u2f.verify_cert", default=True)

I thought I tested it also manually in all directions and it seemed to work. But your comment is legit.

Reply:
It probably doesn't ever happen that "u2f.verify_cert" is not in ``param`` because the policy function sets "u2f.verify_cert" in any case. But we should probably use ``optional=True, default=True`` here anyway.

Reply:
pushed it